src/coalesce-vue-vuetify3/src/components/input/c-datetime-picker.vue

```diff
@@ -76,6 +76,10 @@
           :max="max ? endOfDay(max) : undefined"
           v-bind="datePickerProps"
         >
+          <template v-slot:actions v-if="showTodayButton">
+            <v-btn @click="setToday" :disabled="!isDateAllowed(new Date())"> Today </v-btn>
```

Comment:
Checking if today's date is allowed... is just passing 'new Date()' here sufficient? Is there any worry about time zones?

Reply:
It would certainly be easier to not do this. Leave it up to the developer to just not enable the "Today" button if they know for their use case that selecting today will be out of bounds of their validation rules. 

This validation is happening before the timezone transformations has been applied, so yes, there will be timezone issues here. The other usage of `isDateAllowed` in `emitInput` occurs after timezone transformations.